### ISSUE TITLE
Learning rate for pre-training

### ISSUE THREAD

**Issue by JiaHeng-DLUT** [FoundationVision/Infinity#43]
Awesome work!

I noticed a discrepancy between the paper and the code regarding the learning rate. The paper mentions a learning rate of 6e-5 with a batch size of 4096, but the code uses `args.tblr * total_batch_size / 256` with `args.tblr=6e-3`. Could you clarify the correct setup for pre-training from scratch on 256x256 images?

Thank you!

**Comment by JeyesHan** [comment]
We use a learning rate of 6e-5 with a batch size of 4096. You can adjust lr accordingly.

**Comment by JiaHeng-DLUT** [comment]
Thanks.
